src/python/botan.py

```diff
@@ -837,7 +847,7 @@ def test_certs():
         print("Serial number:     %s" % hex_encode(cert.serial_number()))
         print("Authority Key ID:  %s" % hex_encode(cert.authority_key_id()))
         print("Subject   Key ID:  %s" % hex_encode(cert.subject_key_id()))
-        print("Public key bits:\n%s\n" % hex_encode(cert.subject_public_key_bits()))
+        print("Public key bits:\n%s\n" % b2a_base64(cert.subject_public_key_bits()))
```

Comment:
> I'll take the rest of the PR without this change.

Thank you!

>  in the long run all of this code dies and is replaced by unittest code anyway

But this code serves as a good _usage example_ - I'm not sure unittest would be as usable or as convenient in that role?

> the Python 3 docs say on b2a_base64 'The length of data should be at most 57 to
> adhere to the base64 standard.' which will not be true.

The Python 3 documentation must be untrue:

https://tools.ietf.org/html/rfc4648 that defines Base64 encoding: page 3 says 'MIME enforces a **limit on line length** of **base 64-encoded** data to 76 characters.'  Python 3 docs refer to the older RFC 3548, but it says the same thing: maximum length of the (output) line is limited, not the maximum length of the text to-be-encoded. Python docs seem to refer to some "base64 standard", but the only Base64 standard I know of is RFC 4648 (that superseded RFC 3548).

https://en.wikipedia.org/wiki/Base64 describes Base64, and also talks only about **line** length restriction rather than the entire input data size limit. Examples given in that Wiki article are quite longer than 57 (before encoding :).

The current Botan's cert.toString() (with whose output I'm trying to match the Python's printout) produces Base64-encoded public key that is larger than 57 both before and after encoding...

Having said all that, the final decision of course is yours. 

P.S. I've submitted a bug report to Python team: http://bugs.python.org/issue25495